src/Providers/CloudWatchServiceProvider.php

```diff
@@ -114,11 +114,17 @@ protected function getCredentials()
             throw new IncompleteCloudWatchConfig('Missing region key-value');
         }
 
-        return $awsCredentials = [
+        $awsCredentials = [
             'region' => $cloudWatchConfigs['region'],
             'version' => $cloudWatchConfigs['version'],
-            'credentials' => $cloudWatchConfigs['credentials'],
         ];
+        
+        // If your server has access to cloudwatch through the EC2 role
+        if(array_key_exists('credentials', $cloudWatchConfigs)) {
```

Comment:
@naveenk87 Did you test this? I have not tested this myself. I am asking because it does not seem like this would work
https://github.com/pagevamp/laravel-cloudwatch-logs/blob/bc1136645a51766de384e6b457fda9ed44b2e922/src/Providers/CloudWatchServiceProvider.php#L103-L127
Because we pull the configs from the file 
https://github.com/pagevamp/laravel-cloudwatch-logs/blob/master/config/logging.php
where `credentials` key will always be there in the `config`. The statement you added will always be true and it will just add empty credentials to config.

Reply:
I have tested this by removing the 'credentials' array from `config/logging.php`

Reply:
Can you please update the doc to reflect the changes? 
@naveenk87

Reply:
I have made a change to check for the key. If the key is empty, the credentials is skipped. I think this makes more sense than having to remove the whole credentials array from the config.